assembl/graphql/schema.py

```diff
@@ -46,6 +54,17 @@ def get_node(cls, id, context, info):
 # manually.
 
 
+def get_entries(langstring):
+    # langstring.entries is a backref, it doesn't get updated until the commit.
+    # Even with db.flush(), new entries doesn't show up or a specific entry show up
```

Comment:
`langstring.db.expire(langstring, ['entries'])` solves this issue, changes are in
https://github.com/assembl/assembl/commit/9e4953f57cfb0cc122d6eb9a9a6ac9d76096a398